doc/install_chainer_gpu.rst

```diff
@@ -45,24 +57,7 @@ Install Chainer
 Install Cupy
 ------------
 
-- Add below to your `~/.bashrc`::
-
-    # setup cuda & cudnn
-    export LD_LIBRARY_PATH=/usr/local/lib:/usr/lib:$LD_LIBRARY_PATH
-    export LIBRARY_PATH=/usr/local/lib:/usr/lib:$LIBRARY_PATH
-    export CPATH=/usr/include:$CPATH
-    export CFLAGS=-I/usr/include
-    export LDFLAGS="-L/usr/local/lib -L/usr/lib"
-    if [ -e /usr/local/cuda ]; then
-      export CUDA_PATH=/usr/local/cuda
-      export PATH=$CUDA_PATH/bin:$PATH
-      export CPATH=$CUDA_PATH/include:$CPATH
-      export LD_LIBRARY_PATH=$CUDA_PATH/lib64:$CUDA_PATH/lib:$LD_LIBRARY_PATH
-      export CFLAGS=-I$CUDA_PATH/include
-      export LDFLAGS="-L$CUDA_PATH/lib64 -L$CUDA_PATH/lib"
-    fi
-
-- Install Cupy::
```

Comment:
You need this `CUDA` settings in `~/.bashrc`

Reply:
Why did you remove these lines?

Reply:
Thank you very much for your comment.
I removed these just because I could use chainer with GPU support without writing them in `.bashrc`.
(I thought so because I could execute a sample in #2348.)

I found same explanation in https://docs-cupy.chainer.org/en/latest/install.html.  Sorry for my mistake.
I will make it back to the original.